### PR TITLE
Add parameter to map repetition script

### PR DESCRIPTION
**Pull request by zephyr2k**
@jishnu7 @Bijoshtj  @rohanhc  @rampr Please have a look

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,5 +13,8 @@
     "fs": "0.0.2",
     "jsio": "^2.2.4",
     "underscore": "^1.8.3"
+  },
+  "dependencies": {
+    "path": "^0.12.7"
```
Comment (jishnu7) [inline package.json]: path is built in module node.

Comment (zephyr2k) [inline package.json]: removed

**Comment by rampr** [comment]
LGTM

**Comment by jishnu7** [comment]
:+1:

**Pull request merged**